### PR TITLE
Closure compile index.ts

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,20 +2,25 @@
...
   ],
   "dependencies": {
     "typescript": "^2.3.3"
   },
   "devDependencies": {
     "autoprefixer": "^7.1.1",
+    "google-closure-compiler": "^20170521.0.0",
+    "http-server": "^0.10.0",
     "postcss-cli": "^4.0.0",
-    "http-server": "^0.10.0"
```
Comment (paulirish) [inline package.json]: Deployment on now needs this guy

Comment (paulirish) [inline package.json]: Nvm

File: package.json [hunk 2]
```diff
@@ -2,20 +2,25 @@
...
   "browserslist": [
-     "> 5%"
+    "> 5%"
   ],
   "dependencies": {
     "typescript": "^2.3.3"
   },
   "devDependencies": {
     "autoprefixer": "^7.1.1",
+    "google-closure-compiler": "^20170521.0.0",
```
Comment (tbroyer) [inline package.json]: Any specific reason you're not using [`google-closure-compiler-js`](https://www.npmjs.com/package/google-closure-compiler-js) instead? (that doesn't require Java)

Comment (samccone) [inline package.json]: There are some limitations with that version. 🍇

**Pull request merged**